geppetto-ui/src/3d-canvas/showcase/examples/AuditoryCortexExample.js

```diff
@@ -0,0 +1,209 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core';
+import Canvas from '../../Canvas';
+import model from './model.json';
```

Comment:
This may be moved after the component is rendered (componentDidMount shoud work)